docker/php-fpm/Dockerfile

```diff
@@ -31,4 +31,4 @@ RUN set -eux; \
 COPY ./start-container.sh /usr/local/bin/start-container
 RUN chmod +x /usr/local/bin/start-container
 
-ENTRYPOINT start-container /wsc $WSC_VERSION
+ENTRYPOINT bash start-container /wsc $WSC_VERSION
```

Comment:
Is there a specific reason for this?

Reply:
I got this message with Docker Desktop: `wsc-55-php-fpm-1  | /bin/sh: start-container: not found`. Adding bash removed it.